file_manager/tests/tests_units.py

```diff
@@ -89,7 +89,7 @@ def tearDown(self) -> None:
 
     def test_file_no_exist(self):
         with self.assertRaises(FileDoesNotExistException):
-            FileManager.generate_link(1)
+            FileManager.generate_link(-1)
```

Comment:
why -1?

Reply:
Because the file with such id definitely does not exist